auth-password-store.el

```diff
@@ -45,10 +45,18 @@
 See `auth-source-search' for details on SPEC."
   (cl-assert (or (null type) (eq type (oref backend type)))
              t "Invalid password-store search: %s %s")
-  (when (listp host)
+  (when (and host (listp host))
+    (warn "auth-pass-search ignores all but first host in spec.")
     ;; Take the first non-nil item of the list of hosts
     (setq host (seq-find #'identity host)))
-  (list (auth-pass--build-result host port user)))
+  (cond ((eq host t)
+         (warn "auth-pass-store does not handle host wildcards.")
```

Comment:
`auth-pass-store` -> `auth-password-store` (the user shouldn't have to care about implementation)